client/js/components/GameView/Card.vue

```diff
@@ -22,9 +22,12 @@
       opacity=".8"
     />
     <transition name="slide-above">
-      <v-overlay v-if="scuttledBy" absolute :value="true" class="scuttled-by-overlay">
-        <card :suit="scuttledBy.suit" :rank="scuttledBy.rank"></card>
-      </v-overlay>
+      <template v-if="scuttledBy">
+        <img
+          class="scuttled-by-card"
+          :src="require(`../../img/cards/card_${scuttledBy.suit}_${scuttledBy.rank}.svg`)"
+        />
```

Comment:
switched from an instance of `<card>` to a regular `<img>` like the rest of the card images in this component